### PR TITLE
Russian translation fix

### PR DESCRIPTION
**Pull request by fckn-clown**
I reviewed the entire file and manualy fixed almost all the mistakes.
Feel free to ask any questions and check everything by yourself.

### CODE REVIEW

**Review by NeverCookFirst — COMMENTED** [review]
`dash.getpremium = Получить премиум` - is the right

File: src/main/resources/translate_ru.properties [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
 category.notfound = Страница не найдена
 category.invite = Пригласить Lawliet'a
 category.server = Поддержка
 category.faq = FAQ
 category.feedback = Обратная связь
 category.exception = Ошибка сервера
-category.privacy = Выходные данные & Политика конфиденциальности
-category.featurerequests = Предложить Идеи для Функции
-category.new = Новые предложенные идеи для функции
+category.privacy = Выходные данные & Политика Конфиденциальности
```
Comment (NeverCookFirst) [inline src/main/resources/translate_ru.properties:15]: `Выходные данные & Политика конфиденциальности`

File: src/main/resources/translate_ru.properties [hunk 2]
```diff
@@ -287,56 +287,56 @@ dash.select.empty = <Не установлено>
...
 
 #ERROR
 err.button.home = Вернуться на главную страницу
-err.404.title = 404 страница не найдена
+err.404.title = 404 Страница не найдена
 err.404.des = Запрошенная вами страница не существует. Пожалуйста, убедитесь, что в URL нет ошибки.
-err.exception.title = Уупс!
-err.exception.des = Произошла ошибка в наших серверных системах. Пожалуйста, попробуйте позже!
+err.exception.title = Упс!
+err.exception.des = Произошла серверная ошибка. Пожалуйста, повторите попытке позже!
```
Comment (NeverCookFirst) [inline src/main/resources/translate_ru.properties:338]: `err.exception.des = Произошла серверная ошибка. Пожалуйста, повторите попытку позже!`

File: src/main/resources/translate_ru.properties (hunk 1 above)
Comment (NeverCookFirst) [inline src/main/resources/translate_ru.properties]: `category.privacy = Выходные данные & Политика конфиденциальности`

**Review by NeverCookFirst — CHANGES_REQUESTED** [review]
yes

File: src/main/resources/translate_ru.properties [hunk 3]
```diff
@@ -287,56 +287,56 @@ dash.select.empty = <Не установлено>
...
+devvotes.remindswitch = Держите меня в курсе будущих голосований через ЛС в Discord'е.
+devvotes.reminderupdate.true = Теперь Вы будете получать информацию о будущих голосованиях через личные сообщения в Discord'е. Пожалуйста, не забудь открыть ЛС, чтобы мы могли Вас уведомить.
+devvotes.reminderupdate.false = Вы больше не будете получать информацию о будущих голосованиях через ЛС в Discord'е.
 devvotes.results.slot = {0} ({1} / {2})
 devvotes.results.help = Голосование за этот месяц закончилось, вот результаты:
 
 #ERROR
 err.button.home = Вернуться на главную страницу
-err.404.title = 404 страница не найдена
+err.404.title = 404 Страница не найдена
```
[the remark this answers is not in the document]
Comment (fckn-clown) [reply src/main/resources/translate_ru.properties:335]: Could you please explain this one for me? 
Why there should be a comma and the error code name should be lowercase? 

I mean it's literally the full error code name "404 Страница не найдена" ("404 Not Found") and it always was like that.
If it's necessary then we could maybe change it to something like "Ошибка 404\nСтраница не найдена" ("Error 404\nPage not found").

**Comment by fckn-clown** [comment]
> `dash.getpremium = Получить премиум` - is the right

Could you please explain this one for me?
Why "Premium" in this one should be in lowercase if it's name of the feature?

File: src/main/resources/translate_ru.properties (hunk 3 above)
[the remark this answers is not in the document]
Comment (NeverCookFirst) [reply src/main/resources/translate_ru.properties:335]: > Could you please explain this one for me? Why there should be a comma and the error code name should be lowercase?
> 
> I mean it's literally the full error code name "404 Страница не найдена" ("404 Not Found") and it always was like that. If it's necessary then we could maybe change it to something like "Ошибка 404\nСтраница не найдена" ("Error 404\nPage not found").

согласен

**Comment by NeverCookFirst** [comment]
> > `dash.getpremium = Получить премиум` - is the right
> 
> Could you please explain this one for me? Why "Premium" in this one should be in lowercase if it's name of the feature?

потому что в русском языке такие правила, не пишутся Вот Так Слова, это грубо

**Pull request merged**